staging/src/k8s.io/kubectl/pkg/cmd/util/helpers.go

```diff
@@ -261,7 +261,7 @@ func StandardErrorMessage(err error) (string, bool) {
 	}
 	switch t := err.(type) {
 	case *url.Error:
-		klog.V(4).Infof("Connection error: %s %s: %v", t.Op, t.URL, t.Err)
+		klog.V(4).InfoS("Connection error", t.Op, t.URL, t.Err)
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.